README.md

```diff
@@ -890,6 +891,7 @@ Table of Contents
    * [positionstack](https://positionstack.com/) - Free geocoding for global places and coordinates. 25.000 Requests per month for personal use.
    * [stadiamaps.com](https://stadiamaps.com/) — Map tiles, routing, navigation, and other geospatial APIs. 2,500 free map views and API requests / day for non-commercial usage and testing.
    * [GeocodeAPI](https://geocodeapi.io) - Geocode API: Address to Coordinate Conversion & Geoparsing based on Pelias. Batch geocoding via CSV. 350000 free requests/month.
+   * [BigDataCloud Free Reverse Gecoding API](https://www.bigdatacloud.com/geocoding-apis/free-reverse-geocode-to-city-api) - Free client-side reverse geocoding API that converts geo-coordinates into physical address. Unlimited query for client side implementation. No API key required.
```

Comment:
seems to me these can be combined into one entry really

Reply:
Thanks for the comment. Coz they have multiple tools for a different category, thought it would be best as a different list. But, will merge these 2 into one list. Thanks.